osmaxx-py/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -29,6 +30,12 @@ class ConversionApiClient(RESTApiJWTClient):
     estimated_file_size_url = '/estimate_size_in_bytes/'
     country_base_url = '/country/'
 
+    @staticmethod
+    def _extraction_processing_overdated(progress, extraction_order):
+        return extraction_order.process_start_time and (progress in ['new', 'received', 'started']) and timezone.now() > (
+            extraction_order.process_start_time + settings.OSMAXX.get('EXTRACTION_PROCESSING_TIMEOUT_TIMEDELTA')
+        )
```

Comment:
Giving intermediate results a name (by extracting them to local variables or to functions) might make this logic easier to understand. Candidates for extracting:
* `progress in ['new', 'received', 'started']` (process unfinished)
* `timezone.now() > (extraction_order.process_start_time + settings.OSMAXX.get('EXTRACTION_PROCESSING_TIMEOUT_TIMEDELTA'))` (timeout reached)
* `extraction_order.process_start_time + settings.OSMAXX.get('EXTRACTION_PROCESSING_TIMEOUT_TIMEDELTA')` (process due time)